recipes/cyvcf2/meta.yaml

```
@@ -14,7 +14,8 @@ source:
     - patches/setup.py.patch
 
 build:
-  number: 1
+  number: 0
+  skip: True  # [py < 37]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
  script_env:
    - CFLAGS=-Wno-implicit-function-declaration -Wno-int-conversion  # [osx and py == 38]
```

This silences some errors that appear on osx with python 3.8 (apparently due to XCode being strict):
https://github.com/scikit-image/scikit-image/issues/5051#issuecomment-729795085

These same errors appear as warnings in the Linux builds with python 3.8